compiler/rustc_data_structures/src/transitive_relation.rs

```diff
@@ -322,12 +322,6 @@ impl<T: Clone + Debug + Eq + Hash> TransitiveRelation<T> {
             .collect()
     }
 
-    /// A "best" parent in some sense. See `parents` and
-    /// `postdom_upper_bound` for more details.
-    pub fn postdom_parent(&self, a: &T) -> Option<&T> {
-        self.mutual_immediate_postdominator(self.parents(a))
-    }
```

Comment:
I'm not comfortable with changes to `rustc_data_structures`, the result might be like swiss cheese and very confusing whenever someone may need the removed functionality - cc @nikomatsakis @pnkfelix

Reply:
Oh I see you mentioned that in the PR description, sorry.

Reply:
(sorry, fat-fingered this one; it looks like it's resolved however so reopen if it's still unresolved)

Reply:
This is actually used for tests, I'm going to move it to test-only code shortly.

Reply:
See https://github.com/rust-lang/rust/pull/83197.